webrtc.html

```diff
@@ -2389,6 +2389,58 @@
           attribute is the track that is associated with this
           <code><a>RTCRtpSender</a></code> object.</p>
         </dd>
+
+        <dt>Promise&lt;void&gt; replaceTrack (MediaStreamTrack withTrack)</dt>
```

Comment:
No precedent, just an attempt to clarify that it is the replacer, not the replacee. E.g. instead of, say, replaceTrackWith(track).